ironfish-cli/src/commands/miners/mined.ts

```diff
@@ -123,10 +124,10 @@ export class MinedCommand extends IronfishCommand {
 
     const amount = MathUtils.round(oreToIron(block.minersFee), 2)
 
-    const link = linkText(
-      `https://explorer.ironfish.network/blocks/${block.hash}`,
-      'view in web',
-    )
+    const explorerLink = `https://explorer.ironfish.network/blocks/${block.hash}`
+    const link = supportsHyperlinks.stdout
+      ? linkText(explorerLink, 'view in web')
+      : explorerLink
```

Comment:
Can you just put this code inside of linkText so that it always falls back to the URL when links are not supported?

Reply:
I didn't get second part. Why we should add `stdIn` parameter that is default to true?

Reply:
If you move it into `linkText()`, which stream will you check support for? So linkText needs to be parameterized now to accept a parameter to indicate which stream to check support for. Though, 99% of the time people will use this module with STDOUT, so it's nice to default that new parameter with `true` so you don't need to configure it most of the time.

Reply:
I made the change, merging.

Reply:
Got it! I was confused by `stdIn`. Thanks for explanation